### PR TITLE
[#5141][#5142][#5143] web(ui): Add support for creating, editing, and deleting schema

### PR DESCRIPTION
**Pull request by LauraXia123**
### What changes were proposed in this pull request?
Add support for creating, editing, and deleting schema
image
image
image
image
image
image

### Why are the changes needed?
N/A

Fix: #5141, #5142, #5143

### Does this PR introduce _any_ user-facing change?
N/A

### How was this patch tested?
manually

### CODE REVIEW

File: web/web/src/app/metalakes/metalake/rightContent/tabsContent/tableView/TableView.js [hunk 1]
```diff
@@ -251,31 +262,35 @@ const TableView = () => {
...
+          {!isKafkaSchema && (
+            <IconButton
+              title='Delete'
+              size='small'
+              sx={{ color: theme => theme.palette.error.light }}
+              onClick={() => handleDelete({ name: row.name, type: row.node, catalogType: row.type })}
+              data-refer={`delete-entity-${row.name}`}
+            >
+              <DeleteIcon />
+            </IconButton>
```
Comment (jerryshao) [inline web/web/src/app/metalakes/metalake/rightContent/tabsContent/tableView/TableView.js:292]: Currently, we cannot edit or delete hudi schema, am I right @mchades ?

Comment (mchades) [reply web/web/src/app/metalakes/metalake/rightContent/tabsContent/tableView/TableView.js:292]: yes

Comment (LauraXia123) [reply web/web/src/app/metalakes/metalake/rightContent/tabsContent/tableView/TableView.js:292]: The version of this PR base does not yet support the hudi catalog, I can submit another pr for that later.

File: web/web/src/app/metalakes/metalake/rightContent/RightContent.js [hunk 2]
```diff
@@ -25,23 +25,38 @@ import { Box, Button, IconButton } from '@mui/material'
...
 import TabsContent from './tabsContent/TabsContent'
 import { useSearchParams } from 'next/navigation'
+import { useAppSelector } from '@/lib/hooks/useStore'
 
 const RightContent = () => {
   const [open, setOpen] = useState(false)
+  const [openSchema, setOpenSchema] = useState(false)
   const searchParams = useSearchParams()
   const [isShowBtn, setBtnVisiable] = useState(true)
+  const [isShowSchemaBtn, setSchemaBtnVisiable] = useState(false)
```
Comment (jerryshao) [inline web/web/src/app/metalakes/metalake/rightContent/RightContent.js:38]: I think it should be "Visible", right?

Comment (LauraXia123) [reply web/web/src/app/metalakes/metalake/rightContent/RightContent.js:38]: It's visible when view the schema page, it's unvisible when view other pages. 
The judgment condition is implemented in lines 54 to 57.

Comment (jerryshao) [reply web/web/src/app/metalakes/metalake/rightContent/RightContent.js:38]: I mean "visiable" is a typo we need to fix.

**Pull request merged**